test/e2e/ion-autocomplete.single-select.e2e.html

```diff
@@ -8,47 +8,60 @@
     <script src="../../src/ion-autocomplete.js"></script>
     <script type="text/javascript">
         var app = angular.module('ion-autocomplete-sample', ['ionic', 'ion-autocomplete'])
-                .controller('IonAutocompleteController', function ($scope) {
-
-                    $scope.model = "";
-                    $scope.callbackValueModel = "";
-                    $scope.doneButtonCallbackValueModel = "";
-
-                    $scope.getTestItems = function (query) {
-                        if (query) {
-                            return {
-                                items: [
-                                    {id: "1", name: query + "1", view: "view: " + query + "1"},
-                                    {id: "2", name: query + "2", view: "view: " + query + "2"},
-                                    {id: "3", name: query + "3", view: "view: " + query + "3"}]
-                            };
-                        }
-                        return {items: []};
-                    };
-
-                    $scope.itemsClicked = function (callback) {
-                        $scope.callbackValueModel = callback;
-                    };
-
-                    $scope.doneButtonClicked = function (callback) {
-                        $scope.doneButtonCallbackValueModel = callback;
-                    };
-                });
+            .controller('IonAutocompleteController', function ($scope) {
+
+                $scope.model = "";
+                $scope.callbackValueModel = "";
+                $scope.doneButtonCallbackValueModel = "";
+
+                $scope.getTestItems = function (query) {
+                    if (query) {
+                        return {
+                            items: [
+                                { id: "1", name: query + "1", view: "view: " + query + "1" },
+                                { id: "2", name: query + "2", view: "view: " + query + "2" },
+                                { id: "3", name: query + "3", view: "view: " + query + "3" }]
+                        };
+                    }
+                    return { items: [] };
+                };
+
+                $scope.itemsClicked = function (callback) {
+                    $scope.callbackValueModel = callback;
+                };
+
+                $scope.doneButtonClicked = function (callback) {
+                    $scope.doneButtonCallbackValueModel = callback;
+                };
+            });
     </script>
+    <style>
+        .test-open-class {
+            opacity: 1;
+            bottom: 0;     
+            top:0;          
+        }
+
+        .test-close-class {              
+            top: auto;                      
+            bottom: -100%;            
+        }          
+
+    </style>
 </head>
 
 <body ng-controller="IonAutocompleteController">
-<ion-view title="ion-autocomplete sample">
-    <ion-content>
-        <form>
+    <ion-view title="ion-autocomplete sample">
```

Comment:
Please revert the formatting as this adds unneeded changes to the pull request. Please check the rest of the file as there are more formatting changes like this.